armhf/master_preferences

```diff
@@ -1,4 +1,22 @@
 {
+    "session": {
+        "restore_on_startup": 4,
+        "startup_urls": [
+            "http://start.kano.me/"
+        ]
+    },
+    "distribution": {
+          "import_bookmarks": false,
+          "import_bookmarks_from_file": "/etc/chromium/initial_bookmarks.html",
+          "skip_first_run_ui": true,
+          "make_chrome_default": false,
+          "make_chrome_default_for_user": false,
+          "create_all_shortcuts": true,
+          "show_welcome_page": false
+    },
+    "bookmark_bar": {
+       "show_on_all_tabs": false
+    },
```

Comment:
The rest of this file uses tabs, might be wise to keep to that convention for everything else in this guy.

Reply:
I have noticed that, but the reason why I didn't fix it was to be able to diff the Customisations commit from the original src of much easier. If I had fixed the indentation, the diff would show changes on almost every line in there. Like this, it's clearer what we've added on top of it.

Reply:
Fair enough.